korangar_interface/src/layout/bound.rs

```diff
@@ -172,7 +172,7 @@ impl SizeBound {
     {
         let half_size = size.halved();
         let left = f32::clamp(position.left(), -half_size.width(), available.width() - half_size.width());
-        let top = f32::clamp(position.top(), 0.0, available.height() - 30.0);
+        let top = f32::clamp(position.top(), 0.0, (available.height() - 30.0).max(0.0));
```

Comment:
The game crashes when the available.height() is 0, because in the clamp function the min is 0.0, and the max is -30.0, that trigger the panic and crashes the game.

Reply:
Okay, this `-30.0` seems to be completely wrong anyway. Feel free to remove that :+1: